timely/src/dataflow/channels/pushers/buffer.rs

```diff
@@ -44,7 +44,7 @@ impl<T, CB: Default, P> Buffer<T, CB, P> {
     }
 }
 
-impl<T, C: Container + Data, P: Push<Message<T, C>>> Buffer<T, CapacityContainerBuilder<C>, P> where T: Eq+Clone {
+impl<T, C: SizableContainer + Data, P: Push<Message<T, C>>> Buffer<T, CapacityContainerBuilder<C>, P> where T: Eq+Clone {
```

Comment:
This looks like a new constraint, e.g. as opposed to `IterableContainer`. Is there a reason?

Reply:
I'm confused then. `SizableContainer` is weaker than `Container`? Surely `SizableContainer : Container`?

Reply:
Turns out we don't need this change. Removed.